{"name": "test_login[0702700904-sai_mat_khau-wrong_password]", "status": "passed", "steps": [{"name": "Attempting login with phone: 0702700904, password: sai_mat_khau", "status": "passed", "start": 1754560505842, "stop": 1754560507743}, {"name": "Verify error message for wrong password", "status": "passed", "attachments": [{"name": "Error Message", "source": "6b2b661d-8978-4515-9804-ac40c923d5cb-attachment.txt", "type": "text/plain"}], "start": 1754560507744, "stop": 1754560512359}], "parameters": [{"name": "phone", "value": "'0702700904'"}, {"name": "password", "value": "'sai_mat_khau'"}, {"name": "expected_result", "value": "'wrong_password'"}], "start": 1754560505842, "stop": 1754560512360, "uuid": "a2a01254-4d9e-42fd-8487-0563ac546037", "historyId": "30333fdd9ef09a2ae250c1f99d63ba7d", "testCaseId": "736e2521b0951c3628ea00f9e70dcc7b", "fullName": "tests.test_01_login#test_login", "labels": [{"name": "feature", "value": "Login Functionality"}, {"name": "story", "value": "Test login with valid and invalid credentials"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_01_login"}, {"name": "host", "value": "pnvl161"}, {"name": "thread", "value": "7372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_01_login"}], "titlePath": ["tests", "test_01_login.py"]}